artifacts/4328/deployment.json

```diff
@@ -0,0 +1,7 @@
+{
+	"gasPrice": 1000000000,
+	"gasLimit": 1098052,
```

Comment:
this was not enough gas https://blockexplorer.testnet.avax.boba.network/tx/0xfadf9f16e6be502feffb1cf470c65ea4463082955594b6c6a8f434f0128c6c07

Reply:
Then I think nonce was burn so there's no possibility of deploying the contract

Reply:
and if you re-sign the transaction with a higher nonce?

Reply:
it'd deploy the factory to a different address

Reply:
yep, I guess if that's the case... and we all ack that it should be fine? as long as we have a thread to point people to once we try to merge addresses in safe-deployments repo?

Reply:
also, it's a testnet*

Reply:
No, it shouldn't be fine as addresses will not match the ones expected in safe-deployments

Reply:
Uh uh. Ok. How can we proceed? Ideas?

Reply:
I will generate the deployment again and let's test if somehow the nonce is alright

Reply:
Done, updated the deployment for `chainId=4328`